### PR TITLE
Wiring gui

### PR DESCRIPTION
**Pull request by bxchaney**

### CODE REVIEW

File: gui/dash.py [hunk 1]
```diff
@@ -1,17 +1,70 @@
...
+        in_file = self._input_pane.get_input_path()
+        out_file = self._input_pane.get_output_path()
+
+        try:
+            self._validate_inputs(in_file, out_file)
+        except Exception as e:
+            self._output_pane.setPlainText(e.__str__())
+            return
+     
+        process = subprocess.run([py, parsing_script, in_file, out_file], stdout=subprocess.PIPE)
```
Comment (hbchaney) [inline gui/dash.py]: might be better to run in debug mode here ?

File: gui/input_pane/input.py [hunk 2]
```diff
@@ -1,22 +1,74 @@
...
+        self._go_button = qtw.QPushButton("Go", clicked=go_process)
 
         self._grid = qtw.QGridLayout()
 
-        self._grid.addWidget(self._input_label, 0, 0)
-        self._grid.addWidget(self._input_field, 0, 1)
-        self._grid.addWidget(self._output_folder_label, 1, 0)
-        self._grid.addWidget(self._output_folder_field, 1, 1)
-        self._grid.addWidget(self._button, 2, 0)
+        self._grid.addWidget(self._input_file_label, 0, 0)
```
Comment (hbchaney) [inline gui/input_pane/input.py:27]: can you add more space for seeing the full path by default?

File: gui/dash.py (hunk 1 above)
Comment (bxchaney) [reply gui/dash.py]: fixed

File: gui/dash.py [hunk 3]
```diff
@@ -1,17 +1,73 @@
...
+            return
+     
+        process = subprocess.run(
+            [py, parsing_script, in_file, out_file, "--debug"],
+            stdout=subprocess.PIPE
+        )
+        if process.returncode == 0:
+            out_str = (
+                f"Success!\n"
+                + f"A trimmed version of the .svg file at\n\n{in_file}\n\n"
```
Comment (hbchaney) [inline gui/dash.py:47]:
```suggestion
                + f"A trimmed version of the *.svg file at\n\n{in_file}\n\n"
```

File: gui/input_pane/input.py (hunk 2 above)
Comment (bxchaney) [reply gui/input_pane/input.py:27]: its thicc

**Review by hbchaney — APPROVED** [review]

**Pull request merged**